sdk/formrecognizer/azure-ai-formrecognizer/samples/README.md

```diff
@@ -40,6 +40,7 @@ All of these samples need the endpoint to your Form Recognizer resource ([instru
 |[sample_get_operations.py][sample_get_operations] and [sample_get_operations_async.py][sample_get_operations_async]|Get and list the document model operations created within the past 24 hours|
 |[sample_copy_model.py][sample_copy] and [sample_copy_model_async.py][sample_copy_async]|Copy a custom model from one Form Recognizer resource to another|
 |[sample_get_words_on_document_line.py][sample_get_words_on_document_line] and [sample_get_words_on_document_line_async.py][sample_get_words_on_document_line_async]|Get the words in a DocumentLine|
+|[sample_get_elements_with_spans.py][sample_get_elements_with_spans] and [sample_get_elements_with_spans_async.py][sample_get_elements_with_spans_async]|Get elements in the result of an analyze operation by searching with spans|
```

Comment:
@kristapratico I'm thinking of adding two links for the sync and async samples and adding it into the general table, thoughts? If you also like this, I'll create the links right now :)

Reply:
makes sense to me. Should we move the samples out of the advanced_samples directory now that we're going to pick up telemetry from aka, or do we still plan to try that way as well?

also nit comment - can we give examples of the elements in the description?

Reply:
Done, updated the description. I was thinking of leaving it a little while longer...we are getting some sparse data from there...would like to compare with the stats we get from the links.